docs/Buttons.md

```diff
@@ -244,11 +244,11 @@ export const PostList = () => (
 
 ![Bulk Delete button](./img/bulk-delete-button.png)
 
-| Prop         | Required | Type            | Default            | Description                         |
-| ------------ | -------- | --------------- | ------------------ | ----------------------------------- |
-| `label`      | Optional | `string`        | 'ra.action.delete' | label or translation message to use |
-| `icon`       | Optional | `ReactElement`  | `<DeleteIcon>`     | iconElement, e.g. `<CommentIcon />` |
-| `exporter`   | Optional | `Function`      | -                  | Override the List exporter function |
+| Prop                | Required | Type            | Default            | Description                                        |
+| --------------------| -------- | --------------- | ------------------ | ---------------------------------------------------|
+| `label`             | Optional | `string`        | 'ra.action.delete' | label or translation message to use                |
+| `icon`              | Optional | `ReactElement`  | `<DeleteIcon>`     | iconElement, e.g. `<CommentIcon />`                |
+| `mutationOptions`   | Optional | `object`        | null               | options for react-query `useMutation` hook         |
```

Comment:
The description isn't entirely accurate, as react-query's `options` cannot contain a `meta` key. In fact, you should probably add an example explaining how to set the `meta` parameter.

Reply:
There is a `meta` option in `useMutation`. It is of type `Record<string, unknown>` when ours is `any` though.